tests/test_paperqa.py

```diff
@@ -674,6 +676,33 @@ async def embed_documents(self, texts):
         assert docs.texts_index != docs_shallow_copy.texts_index
         assert docs.texts_index == docs_deep_copy.texts_index
 
+    with subtests.test(msg="clear-vector-store"):
+        # Test that the vector store has content before clearing
+        if isinstance(docs.texts_index, QdrantVectorStore):
+            # For QdrantVectorStore, we need to check if collection exists and has points
+            assert await docs.texts_index._collection_exists()
+            collection_info = await docs.texts_index.client.get_collection(
+                docs.texts_index.collection_name
+            )
+            assert collection_info.points_count > 0
+        else:
+            assert len(docs.texts_index) > 0
+            assert len(docs.texts_index.texts_hashes) > 0
+
+        # Clear the vector store
+        if isinstance(docs.texts_index, QdrantVectorStore):
+            await docs.texts_index.aclear()  # Use aclear() instead of clear()
+        else:
+            docs.texts_index.clear()
```

Comment:
Can we make this `Docs.clear_docs` to be more "full stack":

```suggestion
        # Clear the vector store via Docs
        docs.clear_docs()
```